src/components/card/card.ts

```diff
@@ -34,6 +35,22 @@ export class Card extends LitElement {
   @property({ type: Boolean })
   hideBorder = false;
 
+  /**
+   * Provide a specific `HTML` role for the card based on the usecase (Required to support accessibility).
+   *
+   * **NOTE**: Use the roles which are compatible with the `<a>` tag for `'clickable'` card `'type'`.
+   * */
+  @property({ type: String })
+  cardRole: any = '';
+
+  /** Provides a description for the card (Required to support accessibility).
+   * Example:
+   * For an information card, it can be `'Information card'`.
+   * For an error notification, it can be `'Error notification'`.
+   */
+  @property({ type: String })
+  cardDescription = '';
```

Comment:
These are still defaulting to '' instead of undefined. You can declare them as `varName!: string;`

Reply:
when i declare as  `@property({ type: String }) cardRole!: string;` , getting the following error:
Type 'string' is not assignable to `'"alert" | "alertdialog" | .....`.

So, would it be fine if i declare it as any? `@property({ type: String }) cardRole!: any;`

Reply:
Hmm I'd probably want to dig deeper and figure out why it's trying to type it with specific roles like that.